### DISCUSSION TITLE
Unable to view swagger UI

### DISCUSSION THREAD

**Discussion by akki9270** [magma/magma#5737] [Q&A] [answered]
I have deployed orchestrator 1.4 version on AWS. everything is working fine.
But I'm not able to see swagger ui.

I'm trying to hit this url : https://api.gxcnetwork.com/swagger/v1/ui/

Here are screenshots of my screen and network tab from safari.

<img width="1524" alt="Screenshot 2021-03-25 at 3 39 56 PM" src="https://user-images.githubusercontent.com/22918082/112456101-6a14a400-8d80-11eb-81f0-fe3e82e3697f.png">

![image](https://user-images.githubusercontent.com/22918082/112456657-f9ba5280-8d80-11eb-8f58-2f27754d10c8.png)

how do I get swagger UI on version 1.4 ?

let me know if any other information is required.

**Comment by karthiksubraveti** [comment]
@akki9270 
Our swagger UI implementation is relatively straightforward in 1.4. We are only serving static files here as shown below.
Can you try reaching <orc8r>/apidocs/v1 and check if that works ?
Alternatively can you also try dumping obsidian service logs, perhaps some errors might throw more information on what's going on here. 

Obsidian service plainly points to these folder which is copied when building docker image
ksubraveti@ksubraveti-mbp cloud % grep -ri StaticFolder *
go/obsidian/config.go:	DefaultStaticFolder  = "/var/opt/magma/static"
go/obsidian/config.go:	StaticFolder       string
go/obsidian/server/server.go:	e.Static(obsidian.StaticURLPrefix, obsidian.StaticFolder+"/apidocs")
go/obsidian/server/server.go:	e.Static(obsidian.StaticURLPrefix+"/swagger-ui/dist", obsidian.StaticFolder+"/swagger-ui/dist")
go/obsidian/obsidian/main.go:		&obsidian.StaticFolder, "static_folder", obsidian.DefaultStaticFol

RUN cd /tmp && \
  wget "https://github.com/swagger-api/swagger-ui/archive/v$SWAGGER_UI_VERSION.zip" && \
  unzip "v$SWAGGER_UI_VERSION.zip" -d swagger-ui && \
  mkdir -p /var/opt/magma/static/swagger-ui && \
  cp -r "swagger-ui/swagger-ui-$SWAGGER_UI_VERSION/dist" /var/opt/magma/static/swagger-ui

**Reply by akki9270** [reply]
@karthiksubraveti  it seems there is some problem with obsidian pod. I dumped logs to the attached file and it has one consistent error of missing Network ID, But I don't know what the fix would be.

Please let me know of any suggested way to resolve it. 

[obsidian.log](https://github.com/magma/magma/files/6215403/obsidian.log)

**Comment by hcgatewood** [comment, marked as answer]
Looks like there was a documentation issue. The Swagger UI path for v1.4 is still `/apidocs/v1/`

I've added a PR to reflect those changes in the v1.4 documentation https://github.com/magma/magma/pull/5859. I'll mark this discussion as answered, please follow up if the corrected URL didn't solve

**Comment by akki9270** [comment]
Thanks @hcgatewood the only issue was `/` missing at the end of the url and it worked well.
